### ISSUE TITLE
3.3.7 Autofill entry of Previous Entered Data- Supporting Partner Accessibility Fixes - Level A-  Redundant Entry

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3092]
**User Story**
**As a Supporting Partner with memory/sequencing issues I need the Data I previously entered to be available as an autofill entry so that I don't have to recall it by memory. 

**Context:**
- Theses  issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Supporting Information_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1RsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--401fd26be3124e38981234d207ea635590b5dcd2/2024_Supporting%20Information_Audit.xlsx)
- Information previously entered by or provided to the user that is required to be entered again in the same process is either auto-populated or available for the user to select (except in specific circumstances). Also see 1.3.5 Identify Input Purpose.

**The Problem**
- Form does not save user data, which can pose barriers for users relying on assistive technology or who have memory/sequencing issues who require re-submitting a form.

**Acceptance Criteria**
- [ ] Data previously entered by user should be available as an autofill entry for any user having to resubmit.
<img width="1858" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMnhKQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--32fce51363421a0970fd9b141b20fdf8a9eba423/image.png" alt="image.png" />

**Comment by HRAGANBC** [comment]
@michesmith related to #3110 - can be approached in tandem
